doc/Release_Build_Creation.rst

```diff
@@ -8,19 +8,35 @@ This file consists of the procedure that should be followed to generate the rele
 Prerequisites to setup Virtual Machine
 *****************************************
 
-- Single-Node VM deployment:
+- **Single-Node VM deployment:**
   
   - Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system with the following configuration in Virtual Machine (VM):
-  
-    - RAM: 4GB
-    - CPU: 1 core
-    - NIC: 3
-    - Total Disks: 3
+  - Create VM(s) with at least 4 vCPUs and 4GB of RAM.
+  - Minimum 3 NIC is required on different network (not mandatory)
```

Comment:
`3 NICs are` @TechWriter-Mayur

Reply:
didn't understand the question completely but yes minimum interfaces are 3

Reply:
I think @Saumya-Sunder is pointing that it should be:
Minimum 3 NICs are required on different network.

Please remove not mandatory also. It will only confuse the user.

Reply:
OK